main.go

```diff
@@ -12,27 +12,34 @@ import (
 	"github.com/getlantern/systray"
 )
 
+//go:embed assets/*
+var embeddedFiles embed.FS // Embeds all files in the assets folder
+
 var statusItem, connectionItem *systray.MenuItem
 
 func main() {
-	go func() {
-		systray.Run(onReady, onExit)
-	}()
-	// This keeps the application running in the background.
-	select {}
+	log.Println("Wazuh agent status started...")
+	systray.Run(onReady, onExit)
 }
 
 func onReady() {
 	var iconPath string
 	switch runtime.GOOS {
-	case "linux":
-		iconPath = "./assets/wazuh-logo-min.png"
+	case "linux", "darwin":
+		iconPath = "assets/wazuh-logo-min.png"
 	case "windows":
-		iconPath = "./assets/wazuh-logo-min.ico"
+		iconPath = "assets/wazuh-logo-min.ico"
 	default:
-		iconPath = "./assets/wazuh-logo-min.png" // Fallback to PNG for other OS
+		iconPath = "assets/wazuh-logo-min.png" // Fallback to PNG for other OS
```

Comment:
This means that windows is the only special case.

Reply:
One never knows on which system a user can run this script, but I get the point